Tests/W3CTestSuite/PassingTests.txt

```diff
@@ -279,6 +279,8 @@ __pull_request-551-01.svg
 __pull_request-564-01.svg
 __pull_request-564-02.svg
 __pull_request-575-01.svg
+__pull_request-640-01.svg
```

Comment:
Please add tests in Tests/Svg.UnitTests/PassingTests.csv, too.

Reply:
Fixed